g/github_azure_azure-storage-blob-go/azure-storage-blob-go_v0.10.0_rhel_8.4.sh

```diff
@@ -0,0 +1,76 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : azure-storage-blob-go
+# Version               : v0.10.0
+# Source repo           : https://github.com/Azure/azure-storage-blob-go.git
+# Tested on             : UBI 8.5
+# Language              : GO
+# Travis-Check          : False
```

Comment:
why is this set to false @nailushapotnuru ?

Reply:
corrected, able to execute the script on ubi container.